<!DOCTYPE html>
<html lang="tr">

<! -- header.html -->
<meta charset="utf-8">
<title> C-13-S </title>

<meta name="author" content="Dazhi Jiao">

<!-- Enable responsive viewport -->
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
<!--[if lt IE 9]>
 <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
<![endif]-->

<!-- Le styles -->
<link href="/assets/resources/bootstrap/css/bootstrap.min.css" rel="stylesheet">
<link href="/assets/resources/font-awesome/css/font-awesome.min.css" rel="stylesheet">
<link href="/assets/resources/syntax/syntax.css" rel="stylesheet">
<link href="/assets/css/style.css" rel="stylesheet">

<!-- Le fav and touch icons -->
<!-- Update these with your own images
<link rel="shortcut icon" href="images/favicon.ico">
<link rel="apple-touch-icon" href="images/apple-touch-icon.png">
<link rel="apple-touch-icon" sizes="72x72" href="images/apple-touch-icon-72x72.png">
<link rel="apple-touch-icon" sizes="114x114" href="images/apple-touch-icon-114x114.png">
-->

 <link rel="alternate" type="application/rss+xml" title="" href="/feed.xml">

<! -- header.html end -->


   <body>
   		<nav class="navbar navbar-default visible-xs" role="navigation">
		  <!-- Brand and toggle get grouped for better mobile display -->
		  <div class="navbar-header">
		    <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
			  <span class="sr-only">Toggle navigation</span>
			  <span class="icon-bar"></span>
			  <span class="icon-bar"></span>
			  <span class="icon-bar"></span>
			</button>
			
			<a type="button" class="navbar-toggle nav-link" href="http://github.com/jiaoda">
		      <i class="fa fa-github"></i>
		    </a>
		    
		    
		    <a type="button" class="navbar-toggle nav-link" href="http://twitter.com/djiao">
		      <i class="fa fa-twitter"></i>
		    </a>
		    
		    
		    <a type="button" class="navbar-toggle nav-link" href="mailto:dazhi.jiao@gmail.com">
		      <i class="fa fa-envelope"></i>
		    </a>
		    
			<a class="navbar-brand" href="/">
				<img src="http://www.gravatar.com/avatar/3753f3aa8631c46c0e51525edf9bbbd7?s=35" class="img-circle" />
				C-13-S
			</a>
		  </div>

		  <!-- Collect the nav links, forms, and other content for toggling -->
		  <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
			<ul class="nav navbar-nav">
			  <li class="active"><a href="/">Home</a></li>
			  <li><a href="/categories.html">Categories</a></li>
  			  <li><a href="/tags.html">Tags</a></li>
			</ul>
		  </div><!-- /.navbar-collapse -->
		</nav>

		<div class="col-sm-3 sidebar hidden-xs">
			<! -- sidebar.html -->
<header class="sidebar-header" role="banner">
	<a href="/">
		<img src="http://www.gravatar.com/avatar/3753f3aa8631c46c0e51525edf9bbbd7?s=150" class="img-circle" />
	</a>
	<h3 class="title">
        <a href="/">C-13-S</a>
    </h3>
</header>


<div id="bio" class="text-center">
	PhD Candidate in Informatics
</div>


<div id="contact-list" class="text-center">
	<ul class="list-unstyled list-inline">
		
		<li>
			<a class="btn btn-default btn-sm" href="https://github.com/jiaoda">
				<i class="fa fa-github-alt fa-lg"></i>
			</a>
		</li>
		
		
		<li>
			<a class="btn btn-default btn-sm" href="https://twitter.com/djiao">
				<i class="fa fa-twitter fa-lg"></i>
			</a>
		</li>
		
		
		<li>
			<a class="btn btn-default btn-sm" href="mailto:dazhi.jiao@gmail.com">
				<i class="fa fa-envelope fa-lg"></i>
			</a>
		</li>
		
	</ul>
	<ul id="contact-list-secondary" class="list-unstyled list-inline">
		
		
		
	</ul>
</div>

	

<! -- sidebar.html end -->

		</div>

		<div class="col-sm-9 col-sm-offset-3">
			<div class="page-header">
  <h1> </h1>
</div>



        <article class="home">

            <span class="post-date">
                
                October
                6th,
                
                2009
            </span>

            <h2>
                <a href="/paper%20summary/2009/10/06/summary-drug-discovery-using-chemical-systems-biology-repositioning-the-safe-medicine-comtan-to-treat-multi-drug-and-extensively-drug-resistant-tuberculosis.html">Summary: Drug Discovery Using Chemical Systems Biology: Repositioning the Safe Medicine Comtan to Treat Multi-Drug and Extensively Drug Resistant Tuberculosis</a>
            </h2>

            <div>
                
                    
                        <div>Citation: Kinnings SL, Liu N, Buchmeier N, Tonge PJ, Xie L, Bourne PE. Drug Discovery Using Chemical Systems Biology: Repositioning the Safe Medicine Comtan to Treat Multi-Drug and Extensively Drug Resistant Tuberculosis. PLoS Comput Biol 2009 Jul;5(7):e1000423.<br /><br />This paper is the second paper published by the Bourne and Xie group in "Drug Discovery Using Chemical Systems Biology". It's a very interesting topic: the authors explain how they used chemical systems biology to discover entacapone and tolcapone, commercial available drugs for the treatment of Parkinson't disease, are good candidates for Multi-Drug and Extensively Drug Resistant Tuberculosis (MDR-TB and XDR-TB). These drugs can inhibit the enzyme InhA, which has a similar binding sites with COMT which is their primary target for treatment of the Parkinson's disease. InhA is essential for type II fatty acid biosynthesis and the subsequent synthesis of the bacterial cell wall. It is the common target of the anti-tubercular drugs. Their discoveries are validated by <i>in vitro</i> and InhA kinetic assays using tablets of Comtan, whose active component is entacapone. <br /><br />The authors describe their strategy as follows:<br /><blockquote><br />1. The binding sites of a commercially available drug is extracted or predicted from a 3D structure or model of the target protein. <br />2. Off-targets with similar ligand binding sites are identified across the proteome using an efficient and accurate functional site search algorithm. <br />3. Atomic interactions between the putative off-targets and the drug are evaluated using protein-ligand docking. Only those off-targets that do not experience serious atomic clashes with the drug are selected for further analysis. <br />4. The drug is further optimized to enhance its potency, selectivity and ADME properties by taking into account both the primary target and the off-targets across the genome. <br /></blockquote><br /><br />In short, the authors try to find other targets of a drug in the whole human proteome, based on binding site similarity, and docking. Then they optimize the drug based on all the targets in the proteome. <br /><br />The result is interesting and inspiring. Using this strategy, they found what is described earlier: entacapone and tolcapone, drugs currently in the market for Parkinson's disease can be good candidates for treatment of TB. <br /><br />The authors elaborated on details of their research. The primary target of these drugs is human catehol-<i>O</i>-mehyltransferase (COMT). Using the SOIPPA algorithm, developed by the same group, they detect the common binding sites among proteins. Then they docked entacapone and tolcapone into these proteins and find that InhA were highly ranked. <br /><br />Interestingly, the authors pointed out that when if comparing 2D similarity, these two drugs are not similar the current known InhA inhibitors. When docking 20K drug-like compounds into InhA, entacapone ranked very low. In other words, using virtual screening, they will not be found as potential InhA inhibitors. The logP of these two drugs violates the Linpinski's rule of 5, and is very different from current aanti-tubercular drugs. So these two drugs would be quite unlikely to be selected as lead compounds for the inhibition of InhA, using common drug discovery methods. <br /><br />The authors also compared the difference of the binding poses of the compounds to COMT and InhA, and pointed out possible ways to optimize them so that they can have weaker affinity to the original target COMT. <br /><br />To summarize, this paper successfully presents a case study of using chemical systems biology, in particular using protein-ligand interactions, to assist drug discovery in a new multi-target-multi-drug paradigm.</div>



                    
                
            </div>

        </article>


        <article class="home">

            <span class="post-date">
                
                October
                6th,
                
                2009
            </span>

            <h2>
                <a href="/news/2009/10/06/kao-a-chinese-amrican-share-the-nobel-prize-in-physics.html">Kao, a Chinese Amrican, share the Nobel Prize in Physics</a>
            </h2>

            <div>
                
                    
                        <div><a href='http://www.scientificamerican.com/article.cfm?id=nobel-physics-2009-ccd-fiber-optics'>Nobel Prize in Physics Goes to Pioneer in Fiber Optics and Inventors of Digital Image Sensor [Update]: Scientific American</a><br><blockquote>The 2009 Nobel Prize in Physics will be split among three researchers who laid the groundwork in the 1960s for today's digital-media and telecommunications infrastructure. The Nobel Foundation announced the prizewinners in a news conference this morning from the Royal Swedish Academy of Sciences in Stockholm.<br><br>Half the prize will be awarded to Charles Kao, formerly of Standard Telecommunication Laboratories in Harlow, England (now Nortel), and the Chinese University of Hong Kong. Kao was an early pioneer in the field of fiber optics, the transmission of information over flexible glass fibers.</blockquote></div>



                    
                
            </div>

        </article>


        <article class="home">

            <span class="post-date">
                
                October
                5th,
                
                2009
            </span>

            <h2>
                <a href="/news/2009/10/05/nobel-prize-of-medicine-telomeres-and-telomerase.html">Nobel Prize of Medicine - Telomeres and Telomerase</a>
            </h2>

            <div>
                
                    
                        <div><a href='http://www.scientificamerican.com/article.cfm?id=nobel-prize-medicine-2009-genetics'>Nobel Prize in Medicine shared by three U.S. genetic researchers: Scientific American</a><br><blockquote>The 2009 Nobel Prize in Medicine was awarded this morning in Stockholm to Elizabeth Blackburn, a professor of biology and physiology at the University of California, San Francisco, Carol Greider, a professor in the department of molecular biology and genetics at Johns Hopkins University School of Medicine in Baltimore, and Jack Szostak, a professor of genetics at Massachusetts General Hospital in Boston for the discovery of how chromosomes are protected by telomeres and the enzyme telomerase, the Prize committee announced in a press briefing.<br><br>Telomeres and the enzyme that creates them, telomerase, have had an important impact on the study of cancer, stem cells and aging. </blockquote><br /><a href='http://web.ebscohost.com/ehost/pdf?vid=2&amp;hid=111&amp;sid=b74524aa-7da5-4de1-bcb9-94c2492a00aa%40sessionmgr112'>A nature review published by the three Nobel laureates on telomeres and telomerase. </a><br /><br /><a href='http://www.scientificamerican.com/article.cfm?id=telomeres-telomerase-and'>Telomeres, Telomerase and Cancer</a>, an easier to understand article on Scientific American, by Greider and Blackburn, in 1996.</div>



                    
                
            </div>

        </article>


        <article class="home">

            <span class="post-date">
                
                October
                1st,
                  
                2009
            </span>

            <h2>
                <a href="/fun/2009/10/01/sleep-and-lose-weight.html">Sleep and lose weight</a>
            </h2>

            <div>
                
                    
                        <div>It seems I need to sleep more .<br /><blockquote><p>Lose weight while you sleep? It sounds too good to be true—but recent research indicates that there is a connection between how much you weigh and the amount of shut-eye you get per night.</p>Two hormones, ghrelin and leptin, help to control appetite. When you do not get enough rest, levels of ghrelin, which increases hunger, rise; levels of leptin, which promotes feelings of fullness, sink. <br /></blockquote> <br /><a href='http://www.scientificamerican.com/article.cfm?id=you-snooze-you-lose-weight'>You Snooze, You Lose -- Weight: Scientific American</a><br /><br><br><div class='zemanta-pixie'><img src='http://img.zemanta.com/pixy.gif?x-id=5ee03058-ffff-86df-9caf-691205b2c7e0' alt=''></div></div>



                    
                
            </div>

        </article>


        <article class="home">

            <span class="post-date">
                
                February
                26th,
                
                2009
            </span>

            <h2>
                <a href="/text%20mining/2009/02/26/chemical-name-translation.html">Chemical name translation</a>
            </h2>

            <div>
                
                    
                        <div>A paper published on JCIM discusses issues with manipulating chemical names in various languages, including English, German, Japanese, Chinese, Spanish, Swedish, Polish and Hungarian, in the openeye software lexichem. The method they are using is rule-based. There are a set of rules for each language, to and from English. To translate between two non-english languages, English is used like a lingua franca - for example, to translate between Japanese to Chinese, the chemical name has to be translated from Japanese to English, and then English to Chinese. Details at JCIM <img src='http://img.zemanta.com/pixy.gif?x-id=71d368d1-94e2-439e-8969-dafb971c3c7b'><a href='http://pubs.acs.org/doi/abs/10.1021/ci800243w'>Foreign Language Translation of Chemical Nomenclature by Computer - Journal of Chemical Information and Modeling (ACS Publications)</a><br><br><br><img src='http://pubs.acs.org/appl/literatum/publisher/achs/journals/production/jcisd8/0/jcisd8.ahead-of-print/ci800243w/images/medium/ci-2008-00243w_0053.gif'></div>



                    
                
            </div>

        </article>


        <article class="home">

            <span class="post-date">
                
                February
                6th,
                
                2009
            </span>

            <h2>
                <a href="/2009/02/06/experience-form-mothers-can-be-passed-to-offsprings.html">Experience form mothers can be passed to offsprings</a>
            </h2>

            <div>
                
                    
                        mother's experience before pregnancy in mice can be passed to their offsprings, researches found. Basically, the researchers knockout certain genes (Ras-GRF-2) to cause them having memory defect. Then they put them in an "enriched environment", where they receive a shock to their feet. Then they found that the offsprings of these enriched knockout mice associated the electric shock with the cage.More details at:<a href="http://www.newscientist.com/article/mg20126945.100-can-experiences-be-passed-on-to-offspring.html?DCMP=OTC-rss&amp;nsref=online-news">Can experiences be passed on to offspring? - life - 06 February 2009 - New Scientist</a>



                    
                
            </div>

        </article>


        <article class="home">

            <span class="post-date">
                
                January
                16th,
                
                2009
            </span>

            <h2>
                <a href="/2009/01/16/paper-summary-do-structurally-similar-molecules-have-similar-biological-activity-2.html">Paper summary: "Do Structurally Similar Molecules Have Similar Biological Activity?"</a>
            </h2>

            <div>
                
                    
                        The Martin paper[1] discussed whether similarity of chemical structures could implicate similarity of biological activities. The paper first discussed their findings in previous research led to their conclusion that when designing diverse combinatorial libraries, they would not considering compound that is gt;= 0.85 similar to a compound in their collection. However, other studies had results contradicts it. In this study, the authors used different methods to demonstrate how frequently molecules similar in structure have similar biological activities, and whether they should revise their previous conclusions. They used an IC50 dataset and the MAO dataset to quantify the fraction of chemicals with similar biological activities when they were similar. In their study, they have considered bias in their data, such as not including all active compounds, number of bits in fingerprints. The results showed that at gt;=0.85 Tanimoto similarity in Daylight fingerprints, only 30% of compounds similar to an active compound are active. They came to the conclusion that a redesign of strategies for compound acquisition: not all compounds that have a similarity over 0.85 should be excluded. Small portion of them should be included in the libraries. This paper reviewed many related researches, and clearly demonstrated the prefect results cannot be expected with the statistical nature of library design. However, it doesn’t quantify their suggestions of how much percent of the compounds with gt;=0.85 similarity should be considered and how they should be selected.br/br/1. Martin, Y.C. et al, "Do Structurally Similar Molecules Have    Similar Biological Activity?", iJ. Med. Chem./i, b2002/b,    i45/i, 4350-4358



                    
                
            </div>

        </article>


        <article class="home">

            <span class="post-date">
                
                January
                16th,
                
                2009
            </span>

            <h2>
                <a href="/2009/01/16/paper-review-using-reaction-mechanism-to-measure-enzyme-similarity-2.html">Paper review: "Using Reaction Mechanism to Measure Enzyme Similarity"</a>
            </h2>

            <div>
                
                    
                        The O’Boyle paper described novice quantitative methods to measure the similarities of reactions based upon their mechanisms. The first method was based on fingerprints (FP) calculated from a list of 58 features that described the mechanism steps. After steps were encoded using these features and normalized, they could be compared with similarity comparison. The second method was based on bond changes (BC) occurred in a mechanism step, and the similarity was measured with Tanimoto coefficient. The similarity between two reactions was calculated by pair wise aligning the two mechanisms based on their steps. The results of the two methods displayed reasonable overlaps in finding the most similar enzyme reactions when applied to MACiE dataset. Comparing to the Enzyme Classification (EC), these methods were able to find similarities between reaction mechanisms that appear in different EC classes, as illustrated by a few examples given by the paper. The paper clearly discussed the implication of both the FP and MC methods and explained the features used in the FP methods and described how they can classify chemical reaction mechanisms, as well as what are specific to enzyme catalyzed reactions. I think one interesting discussion could be whether the number of steps (or the complexity of the mechanism) of the mechanism can play a role in the similarity measuring with the two methods. Another extension from this paper might be using these two methods, or especially the features in the FP methods, to cluster the enzymes in the MACiE database and compare the result to the EC classification. br /br /1. O'Boyle, N.M. et al, "Using Reaction Mechanism to Measure Enzyme Similarity", J. Mol. Bio., 2007, 368, 1484-1499 (10.1016/j.jmb.2007.02.065)



                    
                
            </div>

        </article>


        <article class="home">

            <span class="post-date">
                
                November
                11th,
                
                2008
            </span>

            <h2>
                <a href="/2008/11/11/paper-summary-discriminating-between-drugs-and-nondrugs-by-prediction-of-activity-spectra-for-substances-pass-2.html">Paper Summary: "Discriminating between drugs and nondrugs by prediction of activity spectra for substances (PASS)"</a>
            </h2>

            <div>
                
                    
                        The Anzali paper<sup>1</sup> uses PASS (prediction of activity spectra for substances) system to predict drugs and nondrugs. They used the a subset of the World Drug Index (WDI) and Available Chemicals Directory (ADS) as their training set. They first built biological activity spectrum based on SAR analysis of a large data set (&gt; 30k compounds). They then used 5k compounds in WDI and 5k compounds in ADS to train the prediction model based on the biospectra. The method was evaluated using testing set from MDDR. The accuracy of their prediction model is 80% when testing with LOO(leave-one-out). When testing with a dataset of all drugs, the accuracy is 73.4% or 78.5% when the drugs being investigated removed from the testing set. When testing with nondrugs, the accuracy is 83.8%. The authors also showed that when testing with drugs from top-100 prescription pharmaceuticals, 87.5% were correctly predicted. 1. Anzali, S. et al. Discriminating between Drugs and Nondrugs by Prediction of Activity Spectra for Substances (PASS). J. Med. Chem. 44, 2432-2437(2001).



                    
                
            </div>

        </article>


        <article class="home">

            <span class="post-date">
                
                November
                10th,
                
                2008
            </span>

            <h2>
                <a href="/2008/11/10/paper-summary-biological-spectra-analysis-linking-biological-activity-profiles-to-molecular-structure-2.html">Paper summary: "Biological spectra analysis: Linking biological activity profiles to molecular structure"</a>
            </h2>

            <div>
                
                    
                        The Fliri paper<sup>1</sup> discussed a molecular property descriptor that uses biological activity profiles. The descriptor is obtained by using the percent inhibition values at single drug concentration in bioassays that represent a cross section of the proteome, or as named by the authors, biological spectra. The authors uses the percent inhibition values of 1567 structurally diverse compounds and 92 ligand-binding assays from the BioPrint database to construct the spectra. Then they use cosine correlation to calculate the biospectra similarities between compounds. By using the biospectra similarities, they searched the dataset with clotrimazole and tioconazole as entry points. The two compounds were chosen because their target was not in the 92 assays in the biospectra. Both searches returned structurally similar compounds when the similarity ranking is over 0.8. They also compared spectra by hierarchically clustering the datasets by using confidence in cluster similarity. It also identifies structurally similar compounds when CCS values are over 0.8. They authors also used structure information of molecules to predict the biospectra. They added four compounds that were not in the 1567 compounds dataset and reclustered the new 1571 datasets. By comparing the structures of the compounds in the new clusters where the four new compounds were placed, the authors showed that biospectra can be very precise in describing the molecular peroperties and is capable of differentiating molecules on the basis of single atom pair differences. I think activity cliff can also be identified by comparing the biospectra of compounds. However, I am not very clear on how to build such biospectra with new compounds without the assay data. 1. Fliri, A. F., et al.,"Biological Spectra Analysis: Linking Biological Activity Profiles to Molecular Spectra", Proc. Natl. Acad. Sci., 2005, 102, 261-266



                    
                
            </div>

        </article>

<hr/>

<ul class="pager"> 

  
    <li class="previous">
      
        <a href="/page2">&larr; Newer</a>
      
    </li>
  
  
  <li>
          <span class="page_number">Page: 3 of 5</span>
  </li>

  
    <li class="next">
            <a href="/page4">Older &rarr;</a>
    </li>
  

</ul>




			<!-- footer.html -->
<footer>
 <hr/>
 <p>
 	&copy; 2013 Dazhi Jiao with Jekyll. Theme: <a href="https://github.com/dbtek/dbyll">dbyll</a> by dbtek.
 </p>
</footer>

<!-- footer.html end -->


		</div>

    
	<script type="text/javascript" src="/assets/resources/jquery/jquery.min.js"></script>
	<script type="text/javascript" src="/assets/resources/bootstrap/js/bootstrap.min.js"></script>
	<script type="text/javascript" src="/assets/js/app.js"></script>
	<script type="text/javascript">
	  var _gaq = _gaq || [];
	  _gaq.push(['_setAccount', 'UA-7715423-15']);
	  _gaq.push(['_trackPageview']);
	
	  (function() {
	    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
	    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
	    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
	  })();
	</script>
   </body>

</html>


